assets/php/class-wgpb-block-library.php

```diff
@@ -0,0 +1,254 @@
+<?php
+/**
+ * Register the scripts, styles, and blocks needed for the block editor.
+ * NOTE: DO NOT edit this file in WooCommerce core, this is generated from woocommerce-gutenberg-products-block.
+ *
+ * @package WooCommerce\Blocks
+ * @version 3.6.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WGPB_Block_Library Class.
+ */
+class WGPB_Block_Library {
+
+	/**
+	 * Get the file modified time as a cache buster if we're in dev mode.
+	 *
+	 * @param string $file Local path to the file.
+	 * @return string The cache buster value to use for the given file.
+	 */
+	protected static function get_file_version( $file ) {
+		if ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) {
+			$file = trim( $file, '/' );
+			return filemtime( WGPB_ABSPATH . $file );
+		}
+		return WGPB_VERSION;
+	}
+
+	/**
+	 * Registers a script according to `wp_register_script`, additionally loading the translations for the file.
+	 *
+	 * @since 3.6.0
+	 *
+	 * @param string $handle    Name of the script. Should be unique.
+	 * @param string $src       Full URL of the script, or path of the script relative to the WordPress root directory.
+	 * @param array  $deps      Optional. An array of registered script handles this script depends on. Default empty array.
+	 * @param bool   $has_i18n  Optional. Whether to add a script translation call to this file. Default 'false'.
+	 */
+	protected static function register_script( $handle, $src, $deps = array(), $has_i18n = true ) {
+		$filename = str_replace( plugins_url( '/', WGPB_PLUGIN_FILE ), '', $src );
+		$ver      = self::get_file_version( $filename );
+		wp_register_script( $handle, $src, $deps, $ver, true );
+		if ( $has_i18n && function_exists( 'wp_set_script_translations' ) ) {
+			wp_set_script_translations( $handle, 'woo-gutenberg-products-block', WGPB_ABSPATH . 'languages' );
+		}
+	}
+
+	/**
+	 * Registers a style according to `wp_register_style`.
+	 *
+	 * @since 4.1.0
```

Comment:
I'm updating them all to be `2.0.0` (except the Featured Product block, which was introduced in 1.3.0) – it makes sense to me for these to reflect the feature plugin/npm package version, rather than the WC core version.